### PR TITLE
Add subdomains support to WMS and WMTS

### PR DESCRIPTION
**Pull request by gberaudo**
Allows to specify `subdomains` to overcome browser limits on the number of simultaneous requests per host.

See https://cesiumjs.org/forum.html#!topic/cesium-dev/tCCooBxpZFU/discussion.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks @gberaudo and congrats and creating issue #3000!  @kring can you review?

**Comment by gberaudo** [comment]
Any review?

**Comment by pjcozzi** [comment]
I _think_ @kring is on travel; the review may take a bit.

**Comment by gberaudo** [comment]
OK @pjcozzi, I will be patient ;-)

**Comment by kring** [comment]
Yes, sorry.  I'll be back in Australia in a couple of days and will review then.

File: Source/Scene/WebMapServiceImageryProvider.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ define([
      * @constructor
      *
      * @param {Object} options Object with the following properties:
-     * @param {String} options.url The URL of the WMS service.
+     * @param {String} options.url The URL of the WMS service. The URL has the same keywords as the UrlTemplateImageryProvider.
```
Comment (kring) [inline Source/Scene/WebMapServiceImageryProvider.js]: Please make `UrlTemplateImageryProvider` a link, like this:

```
{@link UrlTemplateImageryProvider}
```

**Comment by kring** [comment]
This looks great other than that one minor doc comment, so I'll merge this and make that change myself in master.  Thanks @gberaudo!

**Comment by pjcozzi** [comment]
Thanks again @gberaudo and @kring!

**Comment by gberaudo** [comment]
Thank you for the review @kring.

**Pull request merged**